docs/ovs-offload.md

```diff
@@ -0,0 +1,201 @@
+# OVS Hardware Offload
+
+The OVS software based solution is CPU intensive, affecting system performance
+and preventing fully utilizing available bandwidth. OVS 2.8 and above support
+a feature called OVS Hardware Offload which improves performance significantly.
+This feature allows offloading the OVS data-plane to the NIC while maintaining
+OVS control-plane unmodified. It is using SR-IOV technology with VF representor
+host net-device. The VF representor plays the same role as TAP devices
+in Para-Virtual (PV) setup. A packet sent through the VF representor on the host
+arrives to the VF, and a packet sent through the VF is received by its representor.
+
+## Supported Ethernet controllers
+
+The following manufacturers are known to work:
+
+- Mellanox ConnectX-5 and above
+
+## Prerequisites
+
+- Antrea v0.8.0
+- Linux Kernel 5.7
+- iproute >= 4.12
+- sriov-device-plugin
+- Multus-cni
```

Comment:
Any version requirements there?

Reply:
I wonder if it's confusing to include these 2 here. Sure they are dependencies, but you described their installation in details below.